includes/admin/class-wcmp-export.php

```diff
@@ -1143,14 +1143,40 @@ public function getShipmentData(array $ids, WC_Order $order): array
     }
 
     /**
+     * @param $weight
+     *
+     * @return float
+     */
+    public function getItemWeight_gram($weight): float
```

Comment:
camelcase functie naam graag! En ik mis een param type hier en in de phpdoc voor $weight